src/changelog/unreleased.md

```diff
@@ -57,6 +57,14 @@ changelog entry.
 - Changed `EventLoopProxy::send_event` to `EventLoopProxy::wake_up`, it now
   only wakes up the loop.
 - On Web, slightly improve accuracy of `DeviceEvent::MouseMotion`.
+- `ApplicationHandler::create|destroy_surfaces()` was split off from
```

Comment:
`can_create`

Reply:
Also, since this is plural, I think on Android we'd receive one callback per `Activity`, that its backing `Surface`/`NativeWindow` was created.

(And this may/will happen at different times depending on which of them is visible)

Reply:
I think we can revisit the whole plural thing when multiple activities become actually possible.